### ISSUE TITLE
Different file access permission for different user?

### ISSUE THREAD

**Issue by t-k-** [silverwind/droppy#326]
Is there any way to configure Droppy such that if different user login, they will have different access permissions? If not, is this going to be a future feature?

**Comment by silverwind** [comment]
I don't think that I'll have time or motivation to add something like that. Doing proper permissions that work consistently cross-platforms means I cannot make use file system permissions, but would have to implement everything on my own.

Maybe it's an option to you to run multiple instances for your users instead.

**Comment by t-k-** [comment]
You mentioned you can make use of file system permissions to do what? Is that possible to treat different Droppy users differently by utilizing filesystem permissions?

One ad-hoc approach to solve this issue is to host two or more Droppy instance and using different user config files (with different permissions configured).

**Comment by MatFluor** [comment]
Just tried droppy out. On that note - what is the difference between privileged and normal users?

Would it be possible to just restrict certain actions to non-privileged users? (e.g. they cannot delete etc.)?
